src/diffcalc_API/errors/constraints.py

```diff
@@ -9,18 +9,16 @@
 
 
 class Codes(ErrorCodes):
-    CHECK_CONSTRAINT_EXISTS = 400
+    INVALID_CONSTRAINT_ERROR = 400
 
 
 responses = {code: ALL_RESPONSES[code] for code in np.unique(Codes.all_codes())}
 
 
-def check_constraint_exists(constraint: str) -> None:
-    if constraint not in ALL_CONSTRAINTS:
-        raise DiffcalcAPIException(
-            status_code=Codes.CHECK_CONSTRAINT_EXISTS,
-            detail=(
-                f"property {constraint} does not exist as a valid constraint."
-                f" Choose one of {ALL_CONSTRAINTS}"
-            ),
+class InvalidConstraintError(DiffcalcAPIException):
+    def __init__(self, constraint: str):
+        self.detail = (
+            f"property {constraint} does not exist as a valid constraint."
+            f" Choose one of {ALL_CONSTRAINTS}"
```

Comment:
This wording may read better

```suggestion
            f" Valid constraints are: {ALL_CONSTRAINTS}"
```